packages/api/src/controllers/organicCertifierSurveyController.js

```diff
@@ -259,6 +265,75 @@ const organicCertifierSurveyController = {
+      const plantingManagementPlans = managementPlan.crop_management_plan.planting_management_plans;
+      for (const plantingManagementPlan of plantingManagementPlans) {
+        const location_id = plantingManagementPlan.location_id;
+        !locationIdCropMap[location_id] && (locationIdCropMap[location_id] = new Set());
+      }
+      const hasBeenTransplanted = plantingManagementPlans.filter(plantingManagementPlan => plantingManagementPlan.planting_task_type === 'TRANSPLANT_TASK' && plantingManagementPlan.transplant_task.task.completed_time && new Date(plantingManagementPlan.transplant_task.task.completed_time).getTime() < toDateTime)
+        .sort(({ plant_task: { task: { completed_time: firstCompleteTime } } }, { plant_task: { task: { completed_time: secondCompleteTime } } }) => new Date(secondCompleteTime).getTime() - new Date(firstCompleteTime).getTime())
+        .find(completedTransplantTask => {
+          locationIdCropMap[completedTransplantTask.location_id].add(managementPlan.crop_variety.crop.crop_translation_key);
+          return new Date(completedTransplantTask.transplant_task.task.completed_time).getTime() < fromDateTime;
+        });
+
+      !hasBeenTransplanted && plantingManagementPlans.find(plantingManagementPlan => {
+        if (plantingManagementPlan.planting_task_type === 'PLANT_TASK') {
+          const plantTaskCompleteTime = new Date(plantingManagementPlan.plant_task.task.completed_time).getTime();
+          if (fromDateTime < plantTaskCompleteTime && plantTaskCompleteTime < toDateTime) {
+            locationIdCropMap[plantingManagementPlan.location_id].add(managementPlan.crop_variety.crop.crop_translation_key);
+          }
+          return true;
+        } else if (!plantingManagementPlan.planting_task_type) {
+          plantingManagementPlan.location_id && (locationIdCropMap[plantingManagementPlan.location_id].add(managementPlan.crop_variety.crop.crop_translation_key));
+          return true;
+        }
+        return false;
+      });
+
+      return locationIdCropMap;
+    }, {});
+
+    const locations = await locationModel.query().context({ showHidden: true }).whereNotDeleted()
+      .where({ farm_id })
+      .withGraphJoined(`[
+          figure.[area, line, point], 
+          gate, water_valve, field, garden, buffer_zone, watercourse, fence, 
+          ceremonial_area, residence, surface_water, natural_area,
+          greenhouse, barn, farm_site_boundary
+        ]`);
+
+    const booleanTrueToX = bool => bool ? 'x' : '';
+    return locations.map(location => {
+      const locationOrganicStatus = location[location.figure.type].organic_status;
+      return ({
+        name: location.name,
+        crops: Array.from(locationIdCropMap[location.location_id] || []),
+        area: location.figure?.area?.total_area || location.figure?.line?.total_area || 0,
+        isNew: booleanTrueToX(new Date(location.created_at).getTime() > fromDateTime),
```

Comment:
It's currently possible to generate a spreadsheet where `isNew` (column F) is marked with X, as well as one of G-K. But the text of the form seems to say F-K should be mutually exclusive.